recurly/__init__.py

```diff
@@ -20,6 +20,8 @@
 # You do not want to enable this for production.
 STRICT_MODE = os.getenv("RECURLY_STRICT_MODE", "FALSE").upper() == "TRUE"
 
+DEFAULT_REQUEST_TIMEOUT = os.getenv("RECURLY_REQUEST_TIMEOUT")
```

Comment:
I could see this being

```suggestion
DEFAULT_REQUEST_TIMEOUT = os.getenv("RECURLY_DEFAULT_REQUEST_TIMEOUT")
```

Reply:
Changed this to `RECURLY_DEFAULT_REQUEST_TIMEOUT` to fit the naming scheme